robert_beverage/scripts/generate_qr_code

```diff
@@ -17,23 +38,22 @@ def host_correct(host):
     except (ValueError, AssertionError):
         return False
 
-
+coors = get_coordinates()
 parser = argparse.ArgumentParser(description='generate qr-code')
-parser.add_argument('--x', metavar='x', type=float, nargs=1)
-parser.add_argument('--y', metavar='y', type=float, nargs=1)
 parser.add_argument('--host', metavar='h', type=str, default=["localhost"], nargs=1)
 parser.add_argument('--port', metavar='p', type=int, default=[8080], nargs=1)
 parser.add_argument('--filename', metavar='f', type=str, default=["qr_code.png"], nargs=1)
 args = parser.parse_args()
 
 if not host_correct(args.host[0]):
-    raise Exception("host is not correct")
+    rospy.logerr("host is not correct")
```

Comment:
Incorrect flow of the program. You indicating error and still using wrong Host in QR code generation. You should not do anything else after printing error message